module-system/sov-modules-macros/src/cli_parser.rs

```diff
@@ -67,8 +67,7 @@ impl CliParserMacro {
                     });
 
                     let field_name = field.ident.clone();
-                    let field_name_string = field_name.to_string();
-                    let encode_function_name = format_ident!("encode_{}_call", field_name_string);
+                    let _field_name_string = field_name.to_string();
```

Comment:
Why this variable is underscored?

Reply:
That's a typo, I should have removed it!